main.py

```diff
@@ -109,10 +109,14 @@ async def ranking(msg, *args):
     elif(len(args) == 1 and args[0] == "perfect"):
         await showPerfectRankings(msg.channel)
 
+    elif(len(args) == 3 and args[0] == "swap" and isMemberShift(args[1]) and isMemberShift(args[2])):
```

Comment:
The regex in the `isMemberShift()` function checks for the `+` and `-` symbols. This won't work. The members module might already have a helper function to work with. If it doesn't, you gotta create a helper function for it.